src/crc-start.ts

```diff
@@ -81,7 +81,7 @@ export async function startCrc(
         return true;
       }
     }
-    extensionApi.window.showErrorMessage(err);
+    extensionApi.window.showErrorMessage(`${productName} start error: ${err}`);
```

Comment:
What would the error message be?

In some cases might be multiple lines, right?

Reply:
That is not ideal. We didn't show them on the tray-electron, as it was hard to format.
In that case it might be better to point to the logs.

Also, I believe the issue was more about errors during start that were not shown.

Reply:
The error shown in the issue is due to an update:
https://github.com/crc-org/crc-extension/issues/49#issuecomment-1552617605

In that case we should not show the full error stack, but also believe that was a different issue: #37